1-JavaScript/Week4/challenges/1-sum-entries.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Credit to https://adventofcode.com/ for this exercise
+In the list below you have an array of numbers. 
+The goal is to find the two numbers that add up to 2020.
+Once you have found those numbers, multiply the numbers 
+and store the result of that in the result variable.
+ */
+
+const list = [1721, 979, 366, 299, 675, 1456];
+let result;
+  
+// Write your code here
+for (let i = 0; i< list.length; i++) {
+let a = list[i];
+  let b = 2020 - a;
+    for (let v = 0; v< list.length; v++) {
```

Comment:
There are a couple of small issues in this exercise.

One small improvement could be to change the start point from the second loop. Instead, to start from 0, it could start by i. The reason is that the algorithm already checked if i index has a valid option in the first loop so we don't need to check it again in the second loop.

```
 for (let v = i; v< list.length; v++) {
```

In this case of exercises, where you are looking for something (in this case two numbers that add up to 2020) it is also important to end the loop as soon as you found those numbers.

Could you think about a way to stop looping once you found the result?